
# Plasmid ORFs

::: {.content-hidden unless-profile="ta"}
::: {.callout-note appearance="simple"}
## TA note

Presentation: https://docs.google.com/presentation/d/1amhu1Wrjg_wBtKpSc6C7KZh1dqCUjwDnJlm7dVzpAhA/edit#slide=id.p

Preamble: In general, ORFs can be found to be arbitrarily small. However, the larger the ORF is, the more strongly it indicates a true gene and significant function for it. We can use the standard genetic code, and it should work well since it is from a plasmid and therefore does not have the advanced eukaryotic features.

:::
:::

Gene prediction in bacteria is a crucial step in understanding the genetic makeup of these microorganisms. It involves identifying the locations and boundaries of genes within bacterial genomes, essential for studying bacterial physiology and pathogenicity and developing targeted therapies. The main component of bacterial genes is the open reading frame (ORFs), which translates into protein. So, identifying ORFs is the main task in "de novo" prediction of bacterial genes. The most commonly used start codon is ATG, and less frequently, GTG and TTG. Stop codons are either one of TAA, TAG, or TGA. Although the three "reading frames" on each strand allow ORFs to overlap, this is mainly a feature of highly compacted viral genomes and not often observed in bacteria. Promoter and terminator DNA motifs provide additional information usually built into hidden Markov models or neural networks. Such models also asses the coding potential, codon usage, and length of ORFs to distinguish randomly occurring pairs of start and stop codons from those representing genes.

![NIAID Visual & Medical Arts. 2024-11-08. Filovirus. NIAID BIOART Source. bioart.niaid.nih.gov/bioart/411](images/Plasmid0001-orange.png)

A bacterial plasmid is a circular, double-stranded DNA molecule separate from the genome found in the cytoplasm of bacteria. It is relatively small, ranging from a few kilobases to several hundred kilobases in size, and replicates autonomously, passing to daughter cells at cell division. Although they are usually not essential to the bacterium's survival, plasmids often carry genes that can confer selective advantages to the bacteria under certain conditions. Such conditions include exposure to antibacterial agents like the antibiotics used to combat infections, and some plasmids carry genes making the bacterium resistant to such drugs. 

## Research question

In this exercise, the bacterial DNA you will investigate is a plasmid extracted from an Enterococcus faecium strain to test if it is responsible for its documented resistance to vancomycin. Vancomycin is used to treat infections of bacteria strains resistant to other antibiotics. Methicillin-resistant Staphylococcus aureus (MRSA) is one such strain. Because vancomycin is considered a "last resort" antibiotic, resistance to this drug is dangerous, and it must monitored carefully. 

## Preparation

You will use the [NCBI's ORF Finder tool](https://www.ncbi.nlm.nih.gov/orffinder/) to identify ORFs in the plasmid, and then use various online tools to narrow down the ORFs to a set of likely resistance genes. You can download a fasta file with the plasmid sequence from Brightspace. 

When you open up ORF Finder, start by pasting the fasta file into the "Query Sequence" field. Look at the search parameters, but do not change them yet. One parameter controls the minimum ORF length. We can change the expected genetic code, specify alternate start codons, and choose to ignore nested ORFs.

#### Exercise
Run ORF Finder with the default parameters. 

- How many ORFs did you find?
- What is the longest and shortest ORF in nucleotides (nt)? You can click on "length" to sort them.
- To the right, there is an option to alter the tracks. Try adding the "six-frame translations", which are located under sequence. What is most common, start codons (green) or stop codons (red)?

::: {.content-hidden unless-profile="ta"}
::: {.callout-note appearance="simple"}
## TA note

Number of ORFs: 98

Longest and shortest ORFs found: 1095 and 78

Initiation or termination: Stop codons are more prevalent. This is both due to stop codons having more options, but also due to the natural selection of start codons being restrained to only places wherein expression should happen.

:::
:::

#### Exericse
You can see that a lot of the identified ORFs are tiny and that many of them overlap other ORFs. Both are possible in bacterial plasmids but are much rarer than seen here. Return to the start page and rerun the search, but this time with a minimal ORF size of 300 nucleotides (100 amino acids).

- How many overlapping ORFs do we find now?
- Some of the overlaps may be hard to distinguish visually; you can use the table below the tracks to see the start and stop and order them based on start/stop. Which ORFs overlap?
- There is often a strand bias in transcripted regions. If two genes are close to each other, they need the same strandedness, as RNA polymerases can otherwise collide. Which strand is most common in this plasmid?
- How many genes are on the majority strand?

::: {.content-hidden unless-profile="ta"}
::: {.callout-note appearance="simple"}
## TA note

Example result page after restricting ORF search to 300+ nucleotides

![](images/orf_finding.png)

3 overlaps are found.

ORF13 and ORF1 slightly overlap.
ORF10 and ORF18 greatly overlap.
ORF11 and ORF16 slightly overlap.

+strand is most common. Only two genes are -strand, so 17 genes are +strand. The names are also ordered based on strandedness

17 genes are on the majority strand.

:::
:::

#### Exercise
With this improved list of possible genes, you are now ready to investigate the putative functions of the different ORFs. Sort the ORFs by size and Blast the longest 10. You can also Blast all the ORFs if you want to. You can blast multiple ORFs at the same time. Smartblast is recommended if you want to do it quickly, and it also has multiple useful features. These include showing a phylogeny of the closest related known proteins and a list of the best significant hits. 

Use a table or list, such as the one below, to keep track of your findings. Answer the three questions below for each of the top ten ORFs:

1. Is the translated ORF related to a known protein? Note down the best match with a descriptive name for Putative Function. Sometimes, the best match will be a "hypothetical protein" or similar, in which case you should move on to the next one on the list.

2. What are the functions of the related proteins? Are any of them related to antibiotic resistance, and if so, which antibiotics?

3. The names of the gene homologs you find may themselves suggest resistance function. Go to the [NCBI Gene database](https://www.ncbi.nlm.nih.gov/gene) and search for the gene name to determine if you can find it in a resistance database, such as NDARO. If so, note it as a resistance gene.

| ORF_Label | 	Length |	Putative Function |	Resistance? |
|:---|:---|:---|:---|
|       |       |    |    |
|       |       |    |    |
| ORF18 |	337 | ISL3 family transposase (no strong match) | No |
|       |       |    |    |
|       |       |    |    |
|       |       |    |    |

::: {.content-hidden unless-profile="ta"}
::: {.callout-note appearance="simple"}
## TA note

Table for answering question 1-3 below:
Preamble: When blasting the found ORFs, it is very good if you find a large match, but even just a very good match in part of the ORF also indicates that there is a gene present in the area of the ORF. Smartblast works well for quick answers and a good sampling of possible matches. Make sure that they note down their results in a notebook, file or table.

2.1: All 10 have related known proteins. Depending on your criteria, 2-5 have relatively bad matches, but large regions seem similar, indicating that it is a real protein product.

2.2: Many genes refer to Vancomycin resistance, which is a common antibiotic used against severe infections.

2.3: 7 out of 10 genes are implicated in antibiotic resistance. This does not mean that it is certain they confer resistance (see the transferases), but the genes on plasmid are often there for a reason, strongly indicating that this plasmid confers antibiotic resistance.

2.4: ORF13/ORF1 both have good, resistance-related hits. ORF18 seems plausible, while ORF10 has no hits. ORF16/ORF11 are not the best hits, but both seem possible.

2.5-7: It confers resistance, primarily against Vancomycin, which is a glycopeptide antibiotic used against gram-positive bacteria already resistant to other antibiotics. It is rarely used against Enterococcus nowadays, as they often are resistant.

| ORF_Label | Length | Putative Function | Resistance? | 
|:---|:---|:---|:---|
| ORF8 | 364 | Vancomycin resistance histidine kinase VanS | Yes |
| ORF1 | 343 | Vancomycin/teicoplanin A-type resistance protein VanA  | Yes |
| ORF18 | 337 | ISL3 family transposase (only bad matches) | No |
| ORF13 | 322 | Vancomycin resistance protein VanH  | Yes |
| ORF12 | 302 | Aminoglycoside 6-adenylyltransferase AadK (only bad matches) | Yes |
| ORF11 | 289 | Nucleotidyltransferase domain-containing protein | No |
| ORF17 | 264 | Aminoglycoside O-phosphotransferase APH(3')-IIIa | Yes | 
| ORF3 | 245 | 23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(B) | Yes |
| ORF16 | 244 | class I SAM-dependent methyltransferase | No |
| ORF7 | 231 | Vancomycin resistance response regulator transcription factor VanR-G-Cd | Yes | 
| ORF19 | (11th hit) | 228 | IS6-like element ISSod8 family transposase (only bad matches) | No | 

:::
:::

#### Exercise
Go back to the overlapping ORFs and compare them to the information gathered in your table.

- Which metrics can you use to determine if a hit is good?
- Do both of the overlapping ORFs contain good hits?

::: {.content-hidden unless-profile="ta"}
::: {.callout-note appearance="simple"}
## TA note

E-value, identity, similarity. Identity is not necessarily a good measure for distant homologs.

:::
:::

#### Exercise
Summarize and conclude on your findings.
- Does the plasmid contain genes that confer antibiotic resistance?- If so, what kinds of antibiotics does it provide resistance to?
- Patients hospitalized with infections by MRSA can be treated with Vancomycin. What are the implications, if the patient also happens to have Enterococcus faecium with the plasmid as a part of their gut microbiome?


::: {.content-hidden unless-profile="ta"}
::: {.callout-note appearance="simple"}
## TA note


Treating for MRSA-infections with Vancomycin increases the selection pressure for the other bacteria in the biome. If the resistance genes are present in the patient, these genes become more favorable during treatment. Thereby the risk of the development of Vancomycin resistant strains of Enterococcus faecium is also increased. 

:::
:::


## Project files {.unnumbered}

Download the files you need for this project:

{{< downloadthis ../../project_files/orfproject/orfproject.py dname="orfproject" label="orfproject.py" type="light" >}}
{{< downloadthis ../../project_files/orfproject/test_orfproject.py dname="test_orfproject" label="test_orfproject.py" type="light" >}}
{{< downloadthis ../../project_files/orfproject/e_coli_O157_H157_str_Sakai.fasta dname="e_coli_O157_H157_str_Sakai" label="e_coli_O157_H157_str_Sakai.fasta" type="light" >}}

[https://munch-group.org/bioinformatics/supplementary/project_files](https://munch-group.org/bioinformatics/supplementary/project_files)


